README.md

```diff
@@ -93,6 +95,21 @@ Robusta KRR uses the following Prometheus queries to gather usage data:
 
 _These queries can be customized to suit your specific needs in the future versions, allowing for even more accurate recommendations._
 
+#### Resource Recommendations
+
+The default calculation in Robusta KRR uses a simple strategy to determine resource recommendations based on historical usage data. This strategy takes into account the highest resource usage percentiles for both CPU and memory to provide appropriate recommendations.
```

Comment:
Can you simplify this whole section. It's a lot of words for something very simple. "By default, we give recommendations that match the 99th percentile over the past week. Meaning, with these recommendations, you the request would be sufficient 99% of the time. For the remaining 1% of CPU, we recommend not setting a limit and letting your Pod burst above it."

Also, how is memory the 105th percentile?